google/cloud/testing_util/opentelemetry_matchers.h

```diff
@@ -42,6 +42,12 @@ MATCHER_P(SpanAttributesImpl, matcher,
                                        result_listener);
 }
 
+MATCHER_P(EventAttributesImpl, matcher,
+          ::testing::DescribeMatcher<SpanAttributeMap>(matcher)) {
+  return ::testing::ExplainMatchResult(matcher, arg.GetAttributes(),
+                                       result_listener);
+}
```

Comment:
Q.  The difference between these two appears to be `->` -v- `.`, which seems small compared to `Span` -v- `Event`.  Is that just the way things are?

Reply:
Yeah, so there is only one opentelemetry attribute type, which is shared by spans and events.

I think the right thing to do is to rename these to `OtelAttribute`, `OtelAttributeMap`, etc.

I didn't want to do this just yet, because it affected too many things outside the scope of the PR. In retrospect, it would have been cleaner to do the renaming first, then send this PR.

My inclination is to open an issue to refactor this later. Is that alright with you?

Reply:
SGTM.